.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+
+node_js: '8'
+
+script:
+  - yarn build-react
```

Comment:
Please also build the Preact version.
Maybe use a matrix for that so we have distinct builds.